RSignatureVerifier/Environment.swift

```diff
@@ -1,58 +1,15 @@
 internal protocol EnvironmentSettable {
     var valueNotFound: String { get }
-    var deviceModel: String { get }
-    var osVersion: String { get }
-    var sdkName: String { get }
-    var sdkVersion: String { get }
-    var languageCode: String? { get }
-    var countryCode: String? { get }
     func value(for key: String) -> String?
 }
 
 internal class Environment {
-    static let eTagKey = "com.rakuten.tech.SignatureVerifier.payloadETag"
-
     let bundle: EnvironmentSettable
     var userDefaults = UserDefaults.standard
 
     var subscriptionKey: String {
         bundle.value(for: "RASProjectSubscriptionKey") ?? bundle.valueNotFound
     }
-    var appId: String {
-        bundle.value(for: "RASApplicationIdentifier" as String) ?? bundle.valueNotFound
-    }
-    var appName: String {
-        bundle.value(for: "CFBundleIdentifier" as String) ?? bundle.valueNotFound
-    }
-    var appVersion: String {
-        bundle.value(for: "CFBundleShortVersionString" as String) ?? bundle.valueNotFound
-    }
-    var deviceModel: String {
-        bundle.deviceModel
-    }
-    var osVersion: String {
-        bundle.osVersion
-    }
-    var sdkName: String {
-        bundle.sdkName
-    }
-    var sdkVersion: String {
-        bundle.sdkVersion
-    }
-    var languageCode: String? {
-        bundle.languageCode
-    }
-    var countryCode: String? {
-        bundle.countryCode
-    }
-    var eTag: String? {
-        get {
-            return userDefaults.string(forKey: Environment.eTagKey)
-        }
-        set {
-            userDefaults.set(newValue, forKey: Environment.eTagKey)
-        }
-    }
```

Comment:
Did backend devs mention anything about eTag support?
It would probably be useful cos I don't think the key is likely to be modified on backend.

Reply:
I asked and they said that eTag is not supported